db/seeds/development/evaluation_criteria.seeds.rb

```diff
@@ -4,30 +4,26 @@
   puts 'Seeding evaluation_criteria'
 
   quality_labels = [
-    { 'grade' => 1, 'label' => 'Not Accepted' },
     { 'grade' => 2, 'label' => 'Needs Improvement' },
     { 'grade' => 3, 'label' => 'Meets Expectations' },
     { 'grade' => 4, 'label' => 'Exceeds Expectations' }
```

Comment:
let's make it `1..3`

Reply:
I have removed Needs Improvement and made it 1,2. As we no longer have a failed grade Needs Improvement doesn't sound right